src/actions/play-pause.action.ts

```diff
@@ -53,10 +53,14 @@ export class PlayPauseAction extends DefaultAction<PlayPauseAction> {
         }
         switch (settings?.action.toUpperCase()) {
             case 'PLAY':
-                this.socket.trackPlay();
+                if (!this.playing) {
+                    this.socket.trackPlay();
+                }
                 break;
             case 'PAUSE':
-                this.socket.trackPause();
+                if (this.playing) {
+                    this.socket.trackPause();
+                }
```

Comment:
Those `if`'s shouldn't matter. It could prevent unnecessary requests, but I don't see the need here.

Reply:
The if's shouldn't matter but it appears that there may be a bug in the YTMDesktop API itself where the "trackPlay" command and "trackPause" commands behave as toggles even when they shouldn't. If the track is already playing and you send a second "trackPlay" command to the YTMDesktop app it will pause the song.

Reply:
Also just to clarify. I have v1.13.0 of the YTMusic Desktop app which I think is the latest. It's possible you may have an old version and this bug was introduced in a newer release.